storage/kubernetes/storage.go

```diff
@@ -298,6 +312,25 @@ func (cli *client) ListPasswords() (passwords []storage.Password, err error) {
 	return
 }
 
+func (cli *client) ListConnectors() (connectors []storage.Connector, err error) {
+	var connectorList ConnectorList
+	if err = cli.list(resourceConnector, &connectorList); err != nil {
+		return connectors, fmt.Errorf("failed to list connectors: %v", err)
+	}
+
+	for _, connector := range connectorList.Connectors {
+		c := storage.Connector{
+			ID:     connector.ID,
+			Type:   connector.Type,
+			Name:   connector.Name,
+			Config: connector.Config,
+		}
+		connectors = append(connectors, c)
```

Comment:
nit: since you know the length of `connectorList.Connectors` you can statically allocate this slice.

```
connectors = make([]storage.Connector, len(connectorList.Connectors))
for i, conn := range connectorList.Connectors {
    connectors[i]] = // ...
}
```